commands/build.go

```diff
@@ -90,16 +90,16 @@ func suggestSettingBuilder(logger *logging.Logger) {
 
 func suggestBuilders(logger *logging.Logger) {
```

Comment:
While you're here, could you move `suggestBuilders` to `commands.go`? Now that it's shared, it should go with the other shared functions.

Reply:
Will do.